docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,33 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- TODO Rename this. We're not only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:<!-- TODO State here more clearly why this is useful as a feature -->
```

Comment:
Maybe we could explain first what is the Security Manager role - what it can do, mention that it doesn't map any Git provider permissions - and only after say who can grant/revoke and how.
WDYT?